src/QMCWaveFunctions/Fermion/MultiSlaterDetTableMethod.cpp

```diff
@@ -23,19 +23,18 @@ namespace qmcplusplus
 MultiSlaterDetTableMethod::MultiSlaterDetTableMethod(ParticleSet& targetPtcl,
                                                      std::vector<std::unique_ptr<MultiDiracDeterminant>>&& dets,
                                                      bool use_pre_computing)
-    : WaveFunctionComponent("MultiSlaterDetTableMethod", ""),
-      RatioTimer(*timer_manager.createTimer(ClassName + "::ratio")),
-      offload_timer(*timer_manager.createTimer(ClassName + "::offload")),
-      EvalGradTimer(*timer_manager.createTimer(ClassName + "::evalGrad")),
-      RatioGradTimer(*timer_manager.createTimer(ClassName + "::ratioGrad")),
-      PrepareGroupTimer(*timer_manager.createTimer(ClassName + "::prepareGroup")),
-      UpdateTimer(*timer_manager.createTimer(ClassName + "::updateBuffer")),
-      AccRejTimer(*timer_manager.createTimer(ClassName + "::Accept_Reject")),
-      EvaluateTimer(*timer_manager.createTimer(ClassName + "::evaluate")),
+    : OptimizableObject("multidet_coefs"),
```

Comment:
Unlike Jastrow factor functors. There is only one set of multi det coefficients and thus the name is hard-coded. This is the tag to select this component.
"multidet_coefs" is just bad.
"msd_coefs"? Let me know your preference.

Reply:
Probably it should be "ci_coefs"